src/shapes/Annotation.js

```diff
@@ -290,7 +290,7 @@ define([
             this.labelTexture = dc.gpuResourceCache.resourceForKey(labelKey);
 
             if (!this.labelTexture) {
-                this.labelTexture = dc.textRenderer.createTexture(dc, this.label, labelFont, false);
+                this.labelTexture = dc.textRenderer.renderText(this.label);
```

Comment:
One last thing, I think we should do as you suggested and default enableOutline to true. In Annotation, we should explicitly set and reset the outline property for now. e.g.
                    dc.textRenderer.enableOutline=false;
                    this.labelTexture = dc.textRenderer.renderText(this.label);
                    dc.textRenderer.enableOutline=true;
If we don't do this, the behaviour of the examples will change. As we discussed we'll revisit how this is done when we get to the attributes behaviour.

Reply:
Done. The examples retained their behavior indeed.